content/Installation/Rust.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Rust"
+date: 2019-11-29T15:26:15Z
+draft: false
+weight: 30
+---
+
+Getting started in Rust is easy, if you are using Linux. Otherwise, you probably need to wait for our next implementation, which will support Windows and Mac too.  
+We have published `enzyme` on [crates.io](https://crates.io/crates/enzyme).
+
+So after getting the following dependencies:
+
+- git
+- ninja
+- cmake
+- libssl-dev
+- libclang-dev
+- Rust (rustup) and any nightly toolchain 
+- ~10GB free storage in $HOME/.cache
+
+you can run
+```sh
+cargo install enzyme
+```
+
+This should only take a minute and give you an enzyme-install binary which you can use to build rustc/llvm/enzyme and a bunch of extra things, which might come helpful.  Now you can start the build process by calling
+```bash
+enzyme-install
+```
+It will build everything inside of `$HOME/.cache`, so please have around 10GB available there.
+Also take care, the build process takes around one hour on a recent 8 core cpu.
+
+## Verifying installation
+
+Calling `enzyme-install` will automatically run Enzyme's unit tests after finishing the build process, so you should see something like
+```sh
+Testing Time: 0.63s
+Passed : 299
+Expectedly Failed: 5
+```
+
+You can also run a few basic Rust tests:
+```sh
+git clone https://github.com/rust-ml/oxide-enzyme
+cd oxide-enzyme/example
+cargo enzyme
+```
+You will find your binary slightly more hidden than usual in `target/$TARGET/debug/example`.
+
+## Using Enzyme
+
+In order to play nicely with cargo, the Rust workflow differs somewhat from the [C++](https://enzyme.mit.edu/getting_started/UsingEnzyme/) using Enzyme section.
+First, you need to specify in your `build.rs` file how which function should be differentiated. 
```

Comment:
```suggestion
First, you need to specify in your `build.rs` file how each function should be differentiated. 
```
would be useful to link to an example. Also, is this `build.rs` approach a short-term solution or is it something you envision supporting longer term? I was hoping for a macro that could go in the source, ideally either in the module with the definition or in another module.